pkgs/applications/misc/electrum/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, fetchFromGitHub, python3, python3Packages, zbar, secp256k1
+{ stdenv, fetchurl, fetchFromGitHub, qt5, python3, python3Packages, zbar, secp256k1
```

Comment:
Please don't import the `qt5` package set. Instead call the package with `libsForQt5.callPackage` and reference `wrapQtAppsHook` etc. directly.

Reply:
@worldofpeace, done.